### PR TITLE
Fix: Failing errors

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?

(The CHANGES.md file tracks all the changes that make it to the `main` branch. Add your change to this file in the following format)
- One line description of your PR [#pr_number](Link to your PR)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: tests/e2e/General/UsageTest.php [hunk 1]
```diff
@@ -146,11 +148,20 @@ public function testPrepareStorageStats(array $data): array
...
-                'permission' => 'bucket'
+                'fileSecurity' => false,
+                'permissions' => [
+                    Permission::read(Role::any()),
+                    Permission::create(Role::any()),
+                    Permission::update(Role::any()),
+                    Permission::delete(Role::any()),
+                ],
             ]);
+            \var_dump($res['body']);
```
Comment (christyjacob4) [inline tests/e2e/General/UsageTest.php:164]:
```suggestion
```

File: app/worker.php [hunk 2]
```diff
@@ -93,9 +93,14 @@
...
 }, ['register']);
 
+Server::setResource('pools', function ($register) {
+    return $register->get('pools');
+}, ['register']);
+
 $pools = $register->get('pools');
 $connection = $pools->get('queue')->pop()->getResource();
 $workerNumber = swoole_cpu_num() * intval(App::getEnv('_APP_WORKER_PER_CORE', 6));
+$workerNumber = 1;
```
Comment (christyjacob4) [inline app/worker.php:103]: only one worker? I think we should remove this

File: app/init.php [hunk 3]
```diff
@@ -847,9 +848,12 @@ function (mixed $value) {
 App::setResource('deletes', fn() => new Delete());
 App::setResource('database', fn() => new EventDatabase());
 App::setResource('messaging', fn() => new Phone());
-App::setResource('queueForFunctions', function (Group $pools) {
-    return new Func($pools->get('queue')->pop()->getResource());
+App::setResource('queueConnection', function (Group $pools) {
```
Comment (christyjacob4) [inline app/init.php:851]: why `queueConnection`? is `queue` already being used somewhere?

File: app/config/variables.php [hunk 4]
```diff
@@ -203,7 +203,16 @@
                 'required' => false,
                 'question' => '',
                 'filter' => ''
-            ]
+            ],
+            [
+                'name' => '_APP_CONSOLE_INVITES',
+                'description' => 'This option allows you to disable the invitation of new users on the Appwrite console. When enabled, console users are allowed to invite new users to a project. By default this option is enabled.',
```
Comment (christyjacob4) [inline app/config/variables.php:209]:
```suggestion
                'description' => 'This option allows you to disable the invitation of new users to the Appwrite console. When enabled, console users are allowed to invite new users to a project. By default this option is enabled.',
```

File: .env [hunk 5]
```diff
@@ -62,7 +63,7 @@ _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_MAINTENANCE_RETENTION_SCHEDULES=86400
-_APP_USAGE_TIMESERIES_INTERVAL=60
+_APP_USAGE_TIMESERIES_INTERVAL=20
```
Comment (christyjacob4) [inline .env:66]: why did we reduce this ?

Comment (Meldiron) [reply .env:66]: There is sleep 20 in test and only 1 retry. That would create 40 seconds time for test to pass, leading remaining 20 secs possible to fail test for no reason.
Decreasing this should resolve it.

Also IIRC, this was something like 5 seconds at some point. We can sync with Damodar to see if 20 is fine for development.

File: app/init.php (hunk 3 above)
Comment (Meldiron) [reply app/init.php:851]: Renamed to queue

File: app/worker.php (hunk 2 above)
Comment (Meldiron) [reply app/worker.php:103]: Removed

File: .env (hunk 5 above)
Comment (christyjacob4) [reply .env:66]: Okay 👍

**Review by christyjacob4 — APPROVED** [review]

File: app/controllers/api/health.php [hunk 6]
```diff
@@ -396,10 +398,11 @@
     ->label('sdk.response.code', Response::STATUS_CODE_OK)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_HEALTH_QUEUE)
+    ->inject('queueConnection')
```
Comment (christyjacob4) [inline app/controllers/api/health.php:401]: we need to update the resource here too

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**